lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_accessor :player_name, :symbol, :position
```

Comment:
Please use attr_reader for these attributes. They are not set anywhere after the GameBoard is initialized.